local/modules/Tinymce/Resources/js/tinymce/filemanager/config/config.php

```diff
@@ -5,7 +5,13 @@
 
 $env = 'prod';
 
-require __DIR__ . '/../../../../../../../../core/vendor/autoload.php';
+if (file_exists(__DIR__ . '/../../../../../../../../core/vendor/autoload.php')) {
+    // Symlinked
+    require_once __DIR__ . '/../../../../../../../../core/vendor/autoload.php';
+} else {
```

Comment:
could you add an elseif statement to handle thelia/thelia-project configuration too please ?
There a bootstrap.php file in the root folder.
Thanks !